spring-integration-hazelcast/build.gradle

```diff
@@ -0,0 +1,26 @@
+apply plugin: 'java'
+apply plugin: 'maven'
+
+group = 'org.springframework.integration'
+
+description = 'Spring Integration Hazelcast Support'
+
+sourceCompatibility = 1.7
+targetCompatibility = 1.7
+
+
+
+repositories {
+        
+     maven { url "http://repo.maven.apache.org/maven2" }
+}
+
+dependencies {
+    compile group: 'org.springframework', name: 'spring-context', version:'4.1.5.RELEASE'
+    compile group: 'org.springframework.integration', name: 'spring-integration-core', version:'4.1.2.RELEASE'
+    compile group: 'com.hazelcast', name: 'hazelcast', version:'3.4.1'
+    compile group: 'com.hazelcast', name: 'hazelcast-spring', version:'3.4.1'
+    compile group: 'com.hazelcast', name: 'hazelcast-client', version:'3.4.1'
+    compile group: 'junit', name: 'junit', version:'4.12'
+    testCompile group: 'org.springframework', name: 'spring-test', version:'4.1.5.RELEASE'
```

Comment:
Hi @artembilan,

I updated the gradle configuration and also warnings etc. It should be ok at the moment ;)